kola/tests/kubernetes/controllerInstall.go

```diff
@@ -0,0 +1,795 @@
+package kubernetes
+
+const controllerInstallScript = `#!/bin/bash
+set -e
+
+# List of etcd servers (http://ip:port), comma separated
+export ETCD_ENDPOINTS=
+
+# Specify the version (vX.Y.Z) of Kubernetes assets to deploy
+export K8S_VER=v1.1.3
```

Comment:
similar style to how https://github.com/coreos/mantle/blob/master/kola/tests/flannel/flannel.go templates different flannel backends? can have tests like `"google.kubernetes.coreosbasic.v1.1.2"` `"google.kubernetes.coreosbasic.v1.2"`